src/overlays/actors/ovl_Obj_Y2lift/z_obj_y2lift.h

```diff
@@ -6,8 +6,11 @@
 struct ObjY2lift;
 
 typedef struct ObjY2lift {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x1C];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ u8 unk15C;
+    /* 0x15D */ u8 unk15D;
+    /* 0x15E */ u8 unk15E;
```

Comment:
unk15D and unk15E seem to be boolean, so make it `true` and `false` and make it `!` instead of `== 0`.